src/commons/sideContent/SideContentAutograderCard.tsx

```diff
@@ -18,51 +17,49 @@ type StateProps = {
   index: number;
 };
 
+const renderResult = (value: any) => {
+  /** A class which is the output of the show() function */
+  const ShapeDrawn = (window as any).ShapeDrawn;
+  if (typeof ShapeDrawn !== 'undefined' && value instanceof ShapeDrawn) {
+    return <SideContentCanvasOutput canvas={value.$canvas} />;
+  } else {
+    return stringify(value);
+  }
+};
+
 class SideContentAutograderCard extends React.Component<SideContentAutograderCardProps, {}> {
   public render() {
-    let gradingStatus: string = '';
-
-    const buildErrorString = (errors: SourceError[]) => {
-      return parseError(errors);
-    };
-
-    const renderResult = (value: any) => {
-      /** A class which is the output of the show() function */
-      const ShapeDrawn = (window as any).ShapeDrawn;
-      if (typeof ShapeDrawn !== 'undefined' && value instanceof ShapeDrawn) {
-        return <SideContentCanvasOutput canvas={value.$canvas} />;
-      } else {
-        return stringify(value);
-      }
-    };
+    const cardClasses = ['AutograderCard'];
 
     if (this.props.testcase.result !== undefined || this.props.testcase.errors) {
-      gradingStatus = ' wrong';
-
       if (stringify(this.props.testcase.result) === this.props.testcase.answer) {
-        gradingStatus = ' correct';
+        cardClasses.push('correct');
+      } else {
+        cardClasses.push('wrong');
       }
     }
 
     // Render a placeholder cell in place of the actual testcase data for hidden testcases
     if (this.props.testcase.type === TestcaseTypes.hidden) {
       return (
-        <div className={'AutograderCard' + gradingStatus}>
+        <div className={cardClasses.join(' ')}>
```

Comment:
Should use the classNames function (which we already have)

Reply:
Thanks for the tip - totally missed out on it!